unity/common.mk

```diff
@@ -54,7 +54,7 @@ $(DLL_TARGET): $(CS_SRC) $(EDITOR_DIR) $(BUILD_DIR)
 $(BUILD_DIR):
 	@ mkdir -p "$@"
 
-ifdef $(UNITY_CHANGESET)
+ifdef UNITY_CHANGESET
```

Comment:
This was interpolating `UNITY_CHANGESET`, so it was effectively saying something like `ifdef 4bf1ec4b23c9`, which was always false.